### PR TITLE
Update package versions + github actions

### PR DESCRIPTION
**Pull request by srubin**
- Update version
- Update packages to latest versions, and check `yarn audit`
- Remove travis tests
- Fix unit tests for all platforms
- Run unit tests on all platforms + node 12, 14, 16

### CODE REVIEW

**Comment by marcello3d** [comment]
Minor, but node 10 is out of LTS, so you could remove: https://nodejs.org/en/about/releases/

**Comment by srubin** [comment]
Nice. Will do.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          # Not strictly necessary, but it may prevent rate limit
+          # errors especially on GitHub-hosted macos machines.
+          token: ${{ secrets.GITHUB_TOKEN }}
+        id: setup-ffmpeg
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node_version }}
+          architecture: ${{ matrix.architecture }}
```
Comment (marcello3d) [inline .github/workflows/test.yml:33]: Is this not implied by the System/OS?

**Review by marcello3d — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (srubin) [reply .github/workflows/test.yml:33]: There is a 32 bit windows image available too

**Pull request merged**